src/bloom.cpp

```diff
@@ -212,6 +212,10 @@ bool CBloomFilter::CheckSpecialTransactionMatchesAndUpdate(const CTransaction &t
     case(TRANSACTION_MNHF_SIGNAL):
         // No additional checks for this transaction types
         return false;
+    case(TRANSACTION_ASSET_LOCK):
+    case(TRANSACTION_ASSET_UNLOCK):
+        // TODO asset lock/unlock bloom?
```

Comment:
I think that it's not needed for us to handle bloom for either of these, but @UdjinM6 please provide your thoughts